test/bats/oathkeeper.bats

```diff
@@ -34,6 +34,13 @@ fetch_config_value() {
   fi
 }
 
+# TODO: Refactor when we clean up env variable handling
+if [[ -n "$OATHKEEPER_HOST" ]]; then
```

Comment:
This should be sue in docker compose for

Reply:
Do you mean like this where we simply  ignore the config files?
https://github.com/GaloyMoney/galoy/compare/refactor-oathkeeper-endpoint